crates/sui-core/src/transaction_driver/mod.rs

```diff
@@ -274,12 +274,12 @@ pub fn choose_transaction_driver_percentage() -> u8 {
         }
     }
 
-    // Default to 50% in simtests.
-    if cfg!(msim) {
-        return 50;
-    }
+    // Default to 50% everywhere except mainnet and unit tests.
+    // if !cfg!(test) {
+    //     return 50;
+    // }
 
-    0
+    50
```

Comment:
Just quickly testing to see if CI will pass with it enabled in unit tests as well